sources.go

```diff
@@ -102,3 +103,47 @@ func (s pstateSource) Discover() ([]string, error) {
 
 	return features, nil
 }
+
+////////////////////////////////////////////////////////////////////////////////
+// Network Source
+
+// Implements main.FeatureSource.
+type networkSource struct{}
+
+func (s networkSource) Name() string { return "netid" }
+func (s networkSource) Discover() ([]string, error) {
+	features := []string{}
+	var total_num_vfs int = 0
+	// sysfs of the node mounted as /hostsys inside the pod
```

Comment:
Can we add a single comment which explains what you are doing? Example: `read the network card details from sysfs and determining if SR-IOV is enabled for each of the interfaces.`